.circleci/config.yml

```diff
@@ -7,20 +7,4 @@ jobs:
     working_directory: "/go/src/github.com/zencoder/go-dash"
```

Comment:
We probably don't need this either anymore, go modules don't require the gopath, so we don't really care where circle checks out the project within the image.